modules/33acrossAnalyticsAdapter.js

```diff
@@ -334,21 +349,67 @@ function analyticEventHandler({ eventType, args }) {
     case EVENTS.AUCTION_INIT:
       onAuctionInit(args);
       break;
-    case EVENTS.BID_REQUESTED:
+    case EVENTS.BID_REQUESTED: // set bid.status: 'pending'
       onBidRequested(args);
       break;
+    case EVENTS.BID_TIMEOUT:
+      setBidStatus(getCachedBid(args.auctionId, args.bidId), 'timeout');
+      break;
     case EVENTS.BID_RESPONSE:
       onBidResponse(args);
       break;
-    case EVENTS.BID_WON:
-      onBidWon(args);
+    case EVENTS.BID_REJECTED:
+      setBidStatus(getCachedBid(args.auctionId, args.bidId), 'rejected');
+      break;
+    case EVENTS.NO_BID:
+    case EVENTS.SEAT_NON_BID:
+      setBidStatus(getCachedBid(args.auctionId, args.bidId), 'noBid');
+      break;
+    case EVENTS.BIDDER_ERROR:
+      if (args.bidderRequest && args.bidderRequest.bids) {
+        for (let bid of args.bidderRequest.bids) {
+          setBidStatus(getCachedBid(args.auctionId, bid.bidId), 'error');
+        }
+      }
       break;
     case EVENTS.AUCTION_END:
       onAuctionEnd(args);
       break;
+    case EVENTS.BID_WON: // bid.status: 'targetingSet' | 'rendered' | 'error'
+      onBidWon(args);
+      break;
     default:
       break;
   }
+
+  log.info('::AE::', eventType, locals.cache.auctions);
```

Comment:
what's the purpose of this info message? if it's required, could you please add another unit test that covers it